templates/src-js.js

```diff
@@ -43,21 +43,22 @@ module.exports = answers => {
 		}, {});
 	}
 	/**
-	 * Initialise ${name} component.
+	 * Initialise o-example component/s.
```

Comment:
was this intentional? 
```suggestion
	 * Initialise ${name} component/s.
```

Reply:
yeah, I thought it needed the `/s` as it does either initialise one or many components

Reply:
sorry I meant `o-example` instead of `${name}`